### DISCUSSION TITLE
Update 6.15 - dev.1

### DISCUSSION THREAD

**Discussion by TBog** [TBog/TBLauncher#306] [Pre-Release]
* add Dock pages
* fix keyboard related issues

**Full Changelog**: https://github.com/TBog/TBLauncher/compare/v6.15...v6.15-dev.1

<hr /><em>This discussion was created from the release <a href='https://github.com/TBog/TBLauncher/releases/tag/v6.15-dev.1'>Update 6.15 - dev.1</a>.</em>

**Comment by ltguillaume** [comment]
This version loses the bottom results when scrolling down:
1. Open Search desktop with keyboard
2. Scroll down quickly

Keyboard closes, the bottom results get cut off, because (1) it doesn't reset the scroll position and (2) it doesn't allow you to scroll back.

**Reply by TBog** [reply]
I can't imagine what could cause that. It certainly doesn't happen on my phone.
Could you please make a video?

**Reply by ltguillaume** [reply]
Ugh, I didn't get a notification of your reply, sorry. Here's a video:

https://user-images.githubusercontent.com/4777345/175787608-204bbc5c-a66a-436b-b549-ab4aa25723e0.mp4

Might have something to do with the fact that I set the max. results for initial search results to be what fits exactly on the screen (so no scrolling)?

**Comment by TBog** [comment]
I think it's because
>Might have something to do with the fact that I set the max. results for initial search results to be what fits exactly on the screen (so no scrolling)

I have a fix for this :grinning:

**Reply by ltguillaume** [reply]
Noice 😎
